controllers/providers.ts

```diff
@@ -126,7 +126,7 @@ export const updateProvider = async (req: NextApiRequest, res: NextApiResponse)
             mutation: UPDATE_PROVIDER,
             variables: { ...fields, id },
           })
-          return res.status(204).json({
+          return res.status(200).json({
```

Comment:
Maybe 205 would be better, as you notice when the data changes we can reset UI

Ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/205

Reply:
With 204 or 205 it does not return me the json to be able to do the update in the session storage, I could put 204 or 205 but I would have to call another endpoint to return the updated data of the profile, instead with 200 I can do the update with the response.

Reply:
That shouldn't be a problem

@Eghinner please help him

Reply:
Yes, 204 and 205 are no content HTTP status, so, you can't send a json or text